sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,194 @@
+# Azure Tracing client library for Java
+This package enables distributed tracing across Azure SDK Java libraries through [opentelemetry][opentelemetry]. OpenTelemetry is an open source, vendor-agnostic, single distribution of libraries to provide metrics collection and distributed tracing for services. 
```

Comment:
```suggestion
This package enables distributed tracing across Azure SDK Java libraries through [opentelemetry][OpenTelemetry]. OpenTelemetry is an open source, vendor-agnostic, single distribution of libraries to provide metrics collection and distributed tracing for services. 
```

Reply:
Whichever is the shown text should use the correct capitalization for the product.

Reply:
updated to use `OpenTelemetry`